17:19:02 **** Build of configuration Debug for project 123 ****
make all 
Cannot run program "make": Launching failed

Error: Program "make" not found in PATH
PATH=[C:/Program Files/Java/jre1.8.0_131/bin/server;C:/Program Files/Java/jre1.8.0_131/bin;C:/Program Files/Java/jre1.8.0_131/lib/amd64;C:\Python27\Lib\site-packages\PyQt4;C:\ProgramData\Oracle\Java\javapath;D:\Program Files\Python36-32\Scripts\;D:\Program Files\Python36-32\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Users\scott\.dnx\bin;C:\Program Files\Microsoft DNX\Dnvm\;C:\Program Files\Microsoft SQL Server\120\Tools\Binn\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\WIDCOMM\Bluetooth Software\;C:\Program Files\WIDCOMM\Bluetooth Software\syswow64;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;D:\MinGW\bin;D:\Program Files\Skype\Phone\;C:\Program Files\Git\cmd;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files\dotnet\;D:\Program Files\doxygen\bin;D:\Program Files\LLVM\bin;D:\Program Files\MATLAB\R2016b\runtime\win64;D:\Program Files\MATLAB\R2016b\bin;D:\Program Files\MATLAB\R2016b\polyspace\bin;C:\Python27;C:\Python27\DLLs;C:\Python27\Scripts;C:\Python27\Lib\site-packages\vtk;C:\Python27\gnuplot\binary;C:\Python27\Lib\site-packages\osgeo;D:\Program Files\pythonxy\SciTE-3.5.1-4;D:\Program Files\pythonxy\console;C:\Program Files (x86)\pythonxy\swig;D:\Program Files\pythonxy\gettext\bin;D:\Program Files\PuTTY\;C:\Users\scott\AppData\Local\Microsoft\WindowsApps;D:\modeltech64_10.4\win64;D:\texlive\2016\bin\win32;D:\Program Files\Microsoft VS Code\bin;D:\modeltech64_10.5\win64;D:\MinGW\bin;;D:\Program Files\Eclipse]

17:19:02 Build Finished (took 110ms)

